sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidgetRenderer.kt

```diff
@@ -14,6 +14,7 @@ internal class BitmapWidgetRenderer(
   private val position: WidgetPosition,
   private val marginX: Float,
   private val marginY: Float,
+  internal val onRendered: (Bitmap) -> Unit,
```

Comment:
Imo it's not really `onRendered` - bitmap is just loaded to the texture, which is rendered after it many times. Maybe rename like `onBitmapLoaded`?

Reply:
Renamed as suggested